### ISSUE TITLE
Database error

### ISSUE THREAD

**Issue by OpenFirstNL** [openstate/open-rechtspraak#77]
Search is currently not yielding results, deeplinks to judges show incorrect information, it shows they are no longer active.

example https://openrechtspraak.nl/person/e8f1b36c-6604-48d7-8620-7026d735b795 - https://namenlijst.rechtspraak.nl/#!/details/Ea0VaXZ88jzQb9qrtWABX1jupJvTCP6K

**Comment by Jonakemon** [comment]
Unfortunately our IP address has been blocked by namenlijst.rechtspraak.nl. Our current logic assumes that a judge that does not exist is a former judge; we never accounted for the situation in which the entire website bocks us. That results in all judges 'not existing' and becoming 'former judges'. 

The logic flaw is located here: https://github.com/openstate/open-rechtspraak/blob/develop/app/scraper/people/enrich_people.py#L38. I'll work on a fix to prevent a block breaking the website, but that won't fix us being blocked whatsoever by namenlijst.rechtspraak.nl. 

@siccovansas is aware of possible fixes to get the site up-and-running again. You should also reach out to the Rechtspraak to see if we can get an unblock.

**Comment by Jonakemon** [comment]
Unfortunately we don't have an update yet, since our IP is still blocked by rechtspraak.nl.

**Comment by Jonakemon** [comment]
Fix was made in #81.
